### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by allanbowe**
## Issue

Updating README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,152 @@ parmcards4;
...
+
+There are three parts to consider:
+
+1. JS request / response
+2. SAS inputs / outputs
+3. Configuration
+
+### JS Request / Response
+
+To install the library you can simply run `npm install sasjs` or include a `<script>` tag with a reference to our [CDN](https://www.jsdelivr.com/package/npm/sasjs).
```
Comment (saadjutt01) [inline README.md:56]: >npm WARN deprecated sasjs@2.20.1: sasjs has been deprecated. Please use https://www.npmjs.com/package/@sasjs/adapter instead.

Comment (saadjutt01) [reply README.md:56]: `npm install @sasjs/adapter`

**Review by saadjutt01 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (krishna-acondy) [reply README.md:56]: JSDelivr link needs to be updated to reflect that as well.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,152 @@ parmcards4;
...
+
+Full technical documentation is available [here](https://adapter.sasjs.io).  The main parts are:
+
+### Instantiation
+The following code will instantiate an instance of the adapter:
+
+```javascript
+let sasJs = new SASjs.default(
```
Comment (krishna-acondy) [inline README.md:64]: If you've installed it via NPM, you can import it as a default import like so:
```
  import SASjs from '@sasjs/adapter';
```

And then you can just instantiate it with `const sasJs = new SASjs({your config})`.

File: README.md (hunk 1 above)
Comment (allanbowe) [reply README.md:56]: good spot!!

File: README.md (hunk 2 above)
Comment (allanbowe) [reply README.md:64]: updated

**Review by krishna-acondy — APPROVED** [review]

**Pull request merged**